We do not have poll.h and since struct pollfd etc. pp is defined in
sys/poll.h we use it instead.

+++ src/noux-pkg/openssh/monitor_fdpass.c
@@ -36,6 +36,10 @@
 #include <errno.h>
 #ifdef HAVE_POLL_H
 #include <poll.h>
+#else
+# ifdef HAVE_SYS_POLL_H
+#  include <sys/poll.h>
+# endif
 #endif
 #include <string.h>
 #include <stdarg.h>
